RobotCode/src/robotVision.cpp

```diff
@@ -39,12 +39,11 @@ RobotVision::RobotVision(int argc, char *argv[]) {
     // Initialize tag detector with options
     tf = tagStandard52h13_create();
 
-
     td = apriltag_detector_create();
     apriltag_detector_add_family(td, tf);
 
     if (errno == ENOMEM) {
-        printf("Unable to add family to detector due to insufficient memory to allocate the tag-family decoder with the default maximum hamming value of 2. Try choosing an alternative tag family.\n");
+        cerr << "Unable to add family to detector due to insufficient memory. Try choosing an alternative tag family.\n";
```

Comment:
we dont want to use GNU libc. Stick with C++ stdlib functions for logging

Reply:
? - cerr is a part of the C++ stdlib.

Reply:
![image](https://github.com/user-attachments/assets/c7deb32d-a53b-49bb-a503-efc07aa649b3)
ye its basically the iostream for the C stderr fd